content/docs/configuration/vault.md

```diff
@@ -464,7 +466,47 @@ needs to talks to Vault.
 Although it is not recommended, you can also use the same Vault role for all of
 your Issuers and ClusterIssuers by omitting the `audience` field and re-using
 the same service account.
-<a name="static-service-account-token"></a>
+
+#### Secretless Authentication with a Service Account (External Vault)
+
```

Comment:
@maelvls I just realized we need to add a little footnote to say this feature is available starting cert-manager 1.15 (just like it says for the above paragraph line 366